src/components/Modals/CreateGnosisSafeModal/index.tsx

```diff
@@ -22,10 +24,13 @@ const CreateGnosisSafeModalContent: FunctionComponent<{
 	const {signer} = useContext(EthersContext)
 	const [processing, setProcessing] = useState(false)
 	const [stage, setStage] = useState<CreateGnosisSafeStage>("chooseOption")
-	const [newGnosis, setNewGnosis] = useState(true)
+	const [newGnosis, setNewGnosis] = useState<boolean | undefined>()
 	const [daoName, setDaoName] = useState("")
+	const [gnosisSafeAddress, setGnosisSafeAddress] = useState("")
+	const [created, setCreated] = useState<"create-success" | "load-success" | "error" | undefined>()
```

Comment:
We usually following `camelCase` notation for such types of constants. So instead of `create-success` and `load-success` it should be `createSuccess` and `loadSuccess`